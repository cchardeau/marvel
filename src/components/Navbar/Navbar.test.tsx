import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import Navbar from './Navbar'

it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<Navbar />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it('renders correctly', () => {
  const output = renderer.create(<Navbar />).toJSON()
  expect(output).toMatchSnapshot()
})
