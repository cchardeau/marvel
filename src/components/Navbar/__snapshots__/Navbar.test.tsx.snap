// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionRelative MuiAppBar-colorPrimary"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <svg
      aria-hidden="true"
      className="MuiSvgIcon-root makeStyles-icon-81"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <circle
        cx="12"
        cy="12"
        r="3.2"
      />
      <path
        d="M9 2L7.17 4H4c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2h-3.17L15 2H9zm3 15c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5z"
      />
    </svg>
    <h6
      className="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit MuiTypography-noWrap"
    >
      Marvel app
    </h6>
  </div>
</header>
`;
